<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adivina el número</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QLabel" name="label_esquina_izquierda">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../Archivo/recursos.qrc">:/Logos/logouat2.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <!-- Tamaño fijo para la imagen izquierda -->
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_esquina_derecha">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../Archivo/recursos.qrc">:/Logos/logouat.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <!-- Tamaño fijo para la imagen derecha -->
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 16px; font-weight: bold; color: #2E86C1;</string>
      </property>
      <property name="text">
       <string>Adivina el número (entre 1 y 100):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_numero">
      <property name="styleSheet">
       <string notr="true">background-color: #EAFAF1; color: #1E8449; padding: 10px; border: 2px solid #1E8449; border-radius: 10px; font-size: 14px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boton_adivinar">
      <property name="styleSheet">
       <string notr="true">background-color: #2E86C1; color: white; padding: 10px; border-radius: 10px; font-size: 14px; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Adivinar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_resultado">
      <property name="styleSheet">
       <string notr="true">font-size: 14px; color: #1E8449; font-weight: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Archivo/recursos.qrc"/>
 </resources>
 <connections/>
</ui>